### PR TITLE
add puppetserver to backup

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Review by ekohl — COMMENTED** [review]
The current backup host is the same machine.

File: puppet/modules/profiles/manifests/puppetserver.pp [hunk 1]
```diff
@@ -14,4 +14,9 @@
   class { 'deploy':
     user => $puppet::server_environments_owner,
   }
+
+  restic::repository { 'puppetserver':
+    backup_cap_dac_read_search => true,
+    backup_path                => '/etc/puppetlabs',
```
[the remark this answers is not in the document]
Comment (evgeni) [reply puppet/modules/profiles/manifests/puppetserver.pp]: added the four paths that seem useful (doesn't seem to be excessive in size either way)

File: puppet/data/common.yaml [hunk 2]
```diff
@@ -20,6 +20,7 @@ profiles::backup::receiver::targets:
   - redmine01
   - master02
```
Comment (ekohl) [inline puppet/data/common.yaml:21]: Slightly related question: shouldn't this be `controller01` now? I may have messed that up in the Jenkins migration.

Comment (evgeni) [reply puppet/data/common.yaml:21]: Most probably it should

**Pull request merged**